config/routes.json

```
@@ -0,0 +1,7 @@
+{
+    "root": "pokemonshowdown.com",
+	"client": "play.pokemonshowdown.com",
+	"dex": "dex.pokemonshowdown.com",
+	"replays": "replay.pokemonshowdown.com",
+	"usercp": "pokemonshowdown.com/users"
```

```suggestion
    "root": "pokemonshowdown.com",
    "client": "play.pokemonshowdown.com",
    "dex": "dex.pokemonshowdown.com",
    "replays": "replay.pokemonshowdown.com",
    "usercp": "pokemonshowdown.com/users"
```

(You had a mix of tabs and spaces here.)